### ISSUE TITLE
Winget Out of Date but can't update

### ISSUE THREAD

**Issue by chan-man** [ChrisTitusTech/winutil#3073]
## Describe the bug
When I try to install chrome for instance via winget I get an error stating winget is out of date. It tries to install and runs into an issue. I'm attaching an image.

## To Reproduce
Run WinUtil
Select Chrome
Install

## Expected behavior
Install Chrome or any other software

## Screenshots
![image](https://github.com/user-attachments/assets/1b272f11-d74d-480e-916b-187d4403cac1)

## Additional context
This is a custom image but have tried it on non custom images as well.

**Comment by chan-man** [comment]
I'm not very versed in PowerShell but I did find a script that has helped me. Feel free to modify as tested and I will as well. I don't think this issue is related to WinUtil but still repairs the issue.
I first ran the following:
` Add-AppxPackage -RegisterByFamilyName -MainPackage Microsoft.DesktopAppInstaller_8wekyb3d8bbwe`
Which may or may not have done anything I'm not sure.
Then ran the following:
`$progressPreference = 'silentlyContinue'
$latestWingetMsixBundleUri = $(Invoke-RestMethod https://api.github.com/repos/microsoft/winget-cli/releases/latest).assets.browser_download_url | Where-Object {$_.EndsWith(".msixbundle")}
$latestWingetMsixBundle = $latestWingetMsixBundleUri.Split("/")[-1]
Write-Information "Downloading winget to artifacts directory..."
Invoke-WebRequest -Uri $latestWingetMsixBundleUri -OutFile "./$latestWingetMsixBundle"
Invoke-WebRequest -Uri https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx -OutFile Microsoft.VCLibs.x64.14.00.Desktop.appx
Add-AppxPackage Microsoft.VCLibs.x64.14.00.Desktop.appx
Add-AppxPackage $latestWingetMsixBundle`

Once I ran this code it fully updated WinGet and I had no further issues.
I would like to note this is only happening on New OS installs for me at the moment. I'm sure the entirety of the code above isn't needed I just don't know what to remove.
